recipe/build_base.bat

```diff
@@ -1,17 +1,33 @@
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+@echo on
 
-ECHO [directories] > mplsetup.cfg
-ECHO basedirlist = %LIBRARY_PREFIX% >> mplsetup.cfg
-ECHO [packages] >> mplsetup.cfg
-ECHO tests = False >> mplsetup.cfg
-ECHO sample_data = False >> mplsetup.cfg
-ECHO toolkits_tests = False >> mplsetup.cfg
-ECHO [libs] >> mplsetup.cfg
-ECHO system_freetype = True >> mplsetup.cfg
+set "MESON_ARGS=%MESON_ARGS% --buildtype=release --prefix=%LIBRARY_PREFIX% --pkg-config-path=%LIBRARY_LIB%\pkgconfig -Dlibdir=lib -Dsystem-freetype=true -Dsystem-qhull=true"
 
-set MPLSETUPCFG=mplsetup.cfg
+if "%CI%" == "azure" (
+    :: Hack to try removing problematic Python from Azure CI image
+    :: Replace with conda-smithy solution when available
+    :: xref: https://github.com/conda-forge/conda-smithy/pull/1966
+    set CLEANUP_DIRS=^
+    C:\hostedtoolcache\windows\Python;^
+    ;
 
-%PYTHON% -m pip install --no-deps --no-build-isolation -vv .
+    mkdir C:\empty
+    for %%f in (%CLEANUP_DIRS:;= %) do (
+        if not [%%f] == [] (
+            echo Removing %%f
+            dir %%f
+            robocopy /purge /r:0 /w:0 /mt /ns /nc /np /nfl /ndl /njh /njs C:\empty %%f > nul 2>&1
+            rmdir /q %%f
+        )
+    )
+    rmdir /q C:\empty
+)
```

Comment:
Am planning to get this into conda-smithy ( https://github.com/conda-forge/conda-smithy/pull/1966 )

However it is taking a bit longer to complete

So not to hold things up, have added similar logic here. With a note to follow up later